sky/skylet/providers/oci/node_provider.py

```diff
@@ -167,6 +167,12 @@ def create_node(self, node_config, tags, count):
         ]
         filters = {tag: tags[tag] for tag in VALIDITY_TAGS if tag in tags}
         running_nodes = self.running_nodes(filters)
+        # Make sure the running nodes (to be reused) has correct tags set, esp.
+        # ray-launch-config. Otherwise the nodes will fail to be reused and
+        # the autoscaler will stop them then start new nodes instead.
+        for running_node_id in running_nodes:
+            self.set_node_tags(running_node_id, tags)
```

Comment:
Nice catch! In SkyPilot, we have disabled the check for the `ray-launch-config`. Can we add the following line in the `oci-ray.yaml` to disable the check of the `ray-launch-config`?
https://github.com/skypilot-org/skypilot/blob/1fc6216692872e0d9e0f5d8d74a839a5becc13e2/sky/templates/aws-ray.yml.j2#L25-L31

Reply:
Thanks Zhanghao @Michaelvll for the suggested fix. It can solve this issue same as my original fix by resetting the node tag. However, there is an issue related to the Autoscaler( #2078 ) which will case a cluster never be started successfully.

Reply:
#2078 is not a bug. I closed it.